sdk/identity/identity-vscode/test/public/node/visualStudioCodeCredential.spec.ts

```diff
@@ -5,8 +5,6 @@
 /* eslint-disable @typescript-eslint/no-require-imports */
 /* eslint-disable sort-imports */
 
-import type { MsalTestCleanup } from "../../../../identity/test/node/msalNodeTestSetup";
-import { msalNodeTestSetup } from "../../../../identity/test/node/msalNodeTestSetup";
 import type { Recorder } from "@azure-tools/test-recorder";
```

Comment:
I did the minimal amount of work here given the current state of vscodecredential (broken for 2+ years and a single test that is skipped with a todo)